.palantir/revapi.yml

```diff
@@ -1365,6 +1365,18 @@ acceptedBreaks:
       justification: "Removing deprecations for 1.10.0"
   "1.10.0":
     org.apache.iceberg:iceberg-core:
+      - code: "java.field.constantValueChanged"
+        old: "field org.apache.iceberg.rest.ResourcePaths.V1_TABLE_SCAN_PLAN"
+        new: "field org.apache.iceberg.rest.ResourcePaths.V1_TABLE_SCAN_PLAN"
+        justification: "Plan API is namespace scoped correct the error"
+      - code: "java.field.constantValueChanged"
+        old: "field org.apache.iceberg.rest.ResourcePaths.V1_TABLE_SCAN_PLAN_SUBMIT"
+        new: "field org.apache.iceberg.rest.ResourcePaths.V1_TABLE_SCAN_PLAN_SUBMIT"
+        justification: "Plan API is namespace scoped correct the error"
+      - code: "java.field.constantValueChanged"
+        old: "field org.apache.iceberg.rest.ResourcePaths.V1_TABLE_SCAN_PLAN_TASKS"
+        new: "field org.apache.iceberg.rest.ResourcePaths.V1_TABLE_SCAN_PLAN_TASKS"
+        justification: "Plan API is namespace scoped correct the error"
```

Comment:
Introduced since iceberg 1.8.0 via https://github.com/apache/iceberg/pull/11756 
not sure if it's we should be doing another release to correct this since post 1.10 because the rest request and parsers are now public, server with java SDK 1.10 might be able to use it to ready with the server side implementation ? 

cc @amogh-jahagirdar  thoughts ?